src/almanack/reporting/cli.py

```diff
@@ -38,14 +38,21 @@ def process_repo_entropy(repo_path: str) -> None:
     print(report_content)
 
     # Return the JSON string and report content
-    return json_string, report_content
+    return json_string
+
+
+def check() -> None:
```

Comment:
Depending on what you think about the comment below, consider renaming this function to something less related to "checking" and more related to "running" or something to do with the formation of the CLI.